### ISSUE TITLE
Vue.js implement?

### ISSUE THREAD

**Issue by loss-and-quick** [velocitatem/garlic#1]
Are there plans for a plugin for vue.js?

**Comment by velocitatem** [comment]
Yes, I tried putting together a custom directive for Nuxt, so that it gets a static export. Currently having some issues with the build rendering.

https://github.com/velocitatem/garlic/blob/garlic-vue/demo-nuxt/plugins/directives.js
